provider/fs/Cargo.toml

```diff
@@ -64,8 +64,6 @@ export = [
     "sha2",
     "icu_provider/datagen",
     "icu_provider/deserialize_json",
-    "icu_provider/deserialize_bincode_1",
```

Comment:
I think we still need these

Reply:
+1, because the dev-dependency enables the features, but not the regular dependency. This should hopefully show up in the features CI build.

Reply:
Thanks, fixed in latest commit 👍